.github/workflows/dev_on_workflow_environment_deploy.yml

```
@@ -26,52 +22,89 @@ jobs:
+          secrets: |-
+            SLACK_BOT_TOKEN:passculture-metier-ehp/passculture-app-native-slack-token
+            SENTRY_AUTH_TOKEN:passculture-metier-ehp/passculture-app-native-sentry-token
+            ANDROID_APPCENTER_API_TOKEN:passculture-metier-ehp/passculture-app-native-android-${{ inputs.ENV }}-token
+            IOS_APPCENTER_API_TOKEN:passculture-metier-ehp/passculture-app-native-ios-${{ inputs.ENV }}-token
+            ANDROID_KEYSTORE_STORE_PASSWORD:passculture-metier-ehp/passculture-app-native-android-${{ inputs.ENV }}-keystore-store-password
+            ANDROID_KEYSTORE_KEY_PASSWORD:passculture-metier-ehp/passculture-app-native-android-${{ inputs.ENV }}-keystore-key-password
+            ANDROID_KEYSTORE:passculture-metier-ehp/passculture-app-native-${{ inputs.ENV }}-keystore
+            ANDROID_GOOGLE_SERVICES_JSON:passculture-metier-ehp/passculture-app-native-android-google-service-json
       - name: 'Render Template'
         id: render_template
         uses: chuhlomin/render-template@v1.6
         with:
-          template: .sentryclirc
+          template: templates_github_ci/.sentryclirc
           vars: |
-            token: ${{ secrets.SENTRY_AUTH_TOKEN }}
-      - uses: ruby/setup-ruby@v1
-        with:
-          bundler-cache: true
+            token: ${{ steps.secrets.outputs.SENTRY_AUTH_TOKEN }}
+          result_path: .sentryclirc
       - name: Deploy soft android for ${{ inputs.ENV }}
         if:  ${{ inputs.TYPE == 'soft' }}
         run: echo ${{ inputs.TYPE }}
           #    run: |
           #      bundle exec fastlane android deploy codepush: --env ${{ inputs.ENV }}
           #    env:
-          #      ANDROID_APPCENTER_API_TOKEN: ${{ secrets.ANDROID_APPCENTER_API_TOKEN }}
+          #      ANDROID_APPCENTER_API_TOKEN: ${{ steps.secrets.outputs.ANDROID_APPCENTER_API_TOKEN }}
       - name: Deploy soft ios App for ${{ inputs.ENV }}
         if:  ${{ inputs.TYPE == 'soft' }}
         run: echo ${{ inputs.TYPE }}
           #    run: |
           #      bundle exec fastlane ios deploy codepush: --env ${{ inputs.ENV }}
           #    env:
-          #      IOS_APPCENTER_API_TOKEN: ${{ secrets.IOS_APPCENTER_API_TOKEN }}
-      - name: Deploy hard android for ${{ inputs.ENV }}
+          #      IOS_APPCENTER_API_TOKEN: ${{ steps.secrets.outputs.IOS_APPCENTER_API_TOKEN }}
+      - name: Create a directory for keystores
         if: ${{ inputs.TYPE == 'hard'}}
-        run: echo ${{ inputs.TYPE }}
-        #     run: |
-        #       bundle exec fastlane android deploy --env "${APP_ENV}" --verbose
-      - name: Deploy hard ios for ${{ inputs.ENV }}
+        run: |
+          mkdir -p android/keystores/
+      - name: Setup android keystore for ${{ inputs.ENV }} environment
+        if: ${{ inputs.TYPE == 'hard'}}
+        run: |
+            echo '${{ steps.secrets.outputs.ANDROID_KEYSTORE }}' |  base64 -di > android/keystores/${{ inputs.ENV }}.keystore
```

afin d'avoir des commandes plus explicite et lisible

si possible (dépend de la distribution)

je t'invite à utiliser les options longues

```suggestion
            echo '${{ steps.secrets.outputs.ANDROID_KEYSTORE }}' |  base64 --decode --input > android/keystores/${{ inputs.ENV }}.keystore
```